codegen-lite/src/test/resources/software/amazon/awssdk/codegen/lite/regions/regions.java

```diff
@@ -19,125 +17,82 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.concurrent.ConcurrentHashMap;
+import software.amazon.awssdk.annotations.Generated;
 import software.amazon.awssdk.annotations.SdkPublicApi;
 import software.amazon.awssdk.utils.Validate;
 
-/**
- * An Amazon Web Services region that hosts a set of Amazon services.
```

Comment:
Can we keep the docs in the generated version?

Reply:
Ah, yes. This was just a miss.